test/containers/trends/SMBGDaysContainer.test.js

```diff
@@ -0,0 +1,88 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2016, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import React from 'react';
+
+import { mount } from 'enzyme';
+
+import * as scales from '../../helpers/scales';
+const {
+  trendsXScale: xScale,
+  trendsYScale: yScale,
+} = scales.trends;
+
+import SMBGDaysContainer
+  from '../../../src/containers/trends/SMBGDaysContainer';
+
+describe('SMBGDaysContainer', () => {
+  let wrapper;
+
+  const props = {
+    data: [
+      { id: '0', value: 120, msPer24: 0, localDate: '2016-08-28' },
+      { id: '1', value: 90, msPer24: 9000000, localDate: '2016-08-28' },
+      { id: '2', value: 180, msPer24: 21600000, localDate: '2016-08-28' },
+    ],
+    grouped: true,
+    lines: true,
+    focusedSmbg: {},
+    focusSmbg: () => {},
+    unfocusSmbg: () => {},
+    xScale,
+    yScale,
+  };
+
+  before(() => {
+    wrapper = mount(<SMBGDaysContainer {...props} />);
+  });
+
+  describe('render', () => {
```

Comment:
Want to add a test re: not rendering anything if `!data`? Same goes for other components if such a test not already there...